### PR TITLE
fix: use vscode.workspace.workspaceFolders

### PR DESCRIPTION
**Pull request by plondon**

### CODE REVIEW

File: extension.ts [hunk 1]
```diff
@@ -13,30 +13,32 @@ function lint(textDocument?: vscode.TextDocument) {
   const files = textDocument
     ? [textDocument.fileName]
     : undefined
-  typeCoverage.lint(vscode.workspace.rootPath!, { files, oldProgram, absolutePath: true }).then((result) => {
```
Comment (plantain-00) [inline extension.ts:16]: `absolutePath: true` is not needed now?

Comment (plondon) [reply extension.ts:16]: TypeScript was throwing an error for absolutePath

Comment (plantain-00) [reply extension.ts:16]: The CI passed for current code, so not an error.

Comment (plondon) [reply extension.ts:16]: Will add it back

**Pull request merged**